frontend/app/src/components/home/BitcoinAccountInfo.svelte

```diff
@@ -41,7 +40,7 @@
 </script>
 
 <div class="account-info">
-    <div class="network-selector">Network: <MultiToggle options={networkOptions} bind:selected={selectedNetwork} /></div>
+    <BitcoinNetworkSelector bind:selectedNetwork={selectedNetwork} />
```

Comment:
There is no functional change to this component, I just extracted the BTC network selector into its own component.